app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,17 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+
+class HomeFragment : WolmoFragment<HomePresenter>() {
+
+    override fun init() { }
+
+    override fun layout(): Int {
```

Comment:
You can do `override fun layout() = R.layout.fragment_home` instead. Move it above `init()`